src/code/z_rcp.c

```diff
@@ -1,143 +1,1430 @@
 #include <ultra64.h>
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_rcp/func_80093370.s")
+Gfx sSetupDL[] = {
+    /* 0x00 */
+    gsDPPipeSync(),
+    gsSPTexture(0xFFFF, 0xFFFF, 0, G_TX_RENDERTILE, G_ON),
```

Comment:
Should we look into fixed point macros?  

```suggestion
    gsSPTexture(qu016(1.0f), qu016(1.0f), 0, G_TX_RENDERTILE, G_ON),
```

Reply:
We don't use fixed point macros in general, and I don't think we should start using them yet since it adds another processing step that makes it harder to compare with the raw command. We may want it eventually but it's too early to introduce it imo.
Also, sm64 and mm projects don't use anything of the sort so we probably don't want to go alone on this.